packages/manager/apps/web/client/app/hosting/translations/Messages_de_DE.json

```diff
@@ -398,13 +398,6 @@
   "hosting_tab_DATABASES_configuration_create_success": "Die Datenbank wird erstellt.",
   "hosting_tab_DATABASES_configuration_create_bdd_adding": "Datenbank wird hinzugefügt",
   "hosting_tab_DATABASES_configuration_create_bdd_added": "Datenbank wurde erstellt",
-  "hosting_tab_DATABASES_configuration_create_upgrade_perso": "Beim \"Basic\" Angebot ist nur eine einzige Datenbank enthalten. Sie können wenn Sie möchten zum Pro oder Performance Angebot wechseln, diese erlauben die Erstellung von bis zu 4 Datenbanken",
```

Comment:
Removed in https://github.com/ovh/manager/commit/2ce4d8c857d5d464608a4b7c41ad100e1a7045ea#diff-eba6276e695a9fee04fbad29b0a0adf1